psycopg/pyproject.toml

```diff
@@ -1,3 +1,102 @@
 [build-system]
 requires = ["setuptools>=49.2.0", "wheel>=0.37"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "psycopg"
+description = "PostgreSQL database adapter for Python"
+version = "3.2.4.dev1"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Topic :: Database",
+    "Topic :: Database :: Front-Ends",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+requires-python = ">= 3.8"
+dependencies = [
+    "backports.zoneinfo >= 0.2.0; python_version < '3.9'",
+    "typing-extensions >= 4.6; python_version < '3.13'",
+    "tzdata; sys_platform == 'win32'",
+]
+
+[[project.authors]]
+name = "Daniele Varrazzo"
+email = "daniele.varrazzo@gmail.com"
+
+[project.license]
+text = "GNU Lesser General Public License v3 (LGPLv3)"
+
+[project.urls]
+Homepage = "https://psycopg.org/"
+Documentation = "https://psycopg.org/psycopg3/docs/"
+Changes = "https://psycopg.org/psycopg3/docs/news.html"
+Code = "https://github.com/psycopg/psycopg"
+"Issue Tracker" = "https://github.com/psycopg/psycopg/issues"
+Download = "https://pypi.org/project/psycopg/"
```

Comment:
What if name it PyPI?  Because it is not only about downloading the package.

Reply:
Is there a standardised set of urls? Sampling some major projects:

- numpy points to https://pypi.org/project/numpy/#files
- Django, SQLAlchemy, setuptools don't have that link.

Considering that the prime place where this link is displayed is the PyPI page, which has a prominent "Download files" link on top of the same RHS column where these links are displayed, I think we can drop it altogether.